### ISSUE TITLE
PN532 as_target change uid

### ISSUE THREAD

**Issue by lraton** [adafruit/Adafruit-PN532#114]
I'm using the  iso14443as_target example to emulate a card, but is possible to change the uid? because if i change  the ppse nothing change. The pn532 is connected via i2c

**Comment by lraton** [comment]
No, i found that the pn change the first byte of the wild with a specific One for avoiding emulation. Probably for this is better using a pn7150

**Comment by maxkarelov** [comment]
@lraton 

> i found that the pn change the first byte of the wild with a specific One for avoiding emulation

I saw that they use 0x08 for the first UID byte. but is it still possible to manage others? example from Seeed-PN532 doesn't work as expected but introduce managing left 3 bytes.

pn7150 dev board like this?

<img width="312" alt="image" src="https://github.com/user-attachments/assets/8e0b3b58-a9fa-4d3f-a3db-a1a0e942aede">

I am started recently to work with emulation and there is no much information, libraries and samples for getting started.
what can you recommend to make it more easy to get first results?

**Comment by lraton** [comment]
I seem to have succeeded, but now I don't remember well that it's been a while since I've had my hands with it and I was considering changing to the 7150. The 7150 Is that from elechouse, it have the same size of 532 but Is purple.
![PN7150_1.jpg](https://github.com/user-attachments/assets/1a3b9b39-fcf8-4c9e-a18f-c3c1e46ed115)

I dont know very much about emulation, i've read some article from Salvador mendoza (https://salmg.net/) that Is the creator of the target function of this library
